nilearn/plotting/displays.py

```diff
@@ -608,22 +608,26 @@ def add_contours(self, img, filled=False, **kwargs):
             If filled=True, contours are displayed with color fillings.
         kwargs:
             Extra keyword arguments are passed to contour, see the
-            documentation of pylab.contour
+            documentation of pylab.contour and see pylab.contourf documentation
+            for arguments related to contours with fillings.
             Useful, arguments are typical "levels", which is a
-            list of values to use for plotting a contour, and
+            list of values to use for plotting a contour or contour
+            fillings (if filled=True), and
             "colors", which is one color or a list of colors for
             these contours.
+            Note: if colors are not specified, default coloring choices
+            (from matplotlib) for contours and contours fillings can be
+            different.
         """
         self._map_show(img, type='contour', **kwargs)
         if filled:
-            colors = kwargs['colors']
-            levels = kwargs['levels']
-            if len(levels) <= 1:
-                # contour fillings levels should be given as (lower, upper).
-                levels.append(np.inf)
-            alpha = kwargs['alpha']
-            self._map_show(img, type='contourf', levels=levels, alpha=alpha,
-                           colors=colors[:3])
+            if 'levels' in kwargs:
```

Comment:
I'd rather not, but we coud do good guesses: + and - 90th percentile and 50th percentile of the absolute value.